src/PhpPact/Consumer/Listener/PactTestListener.php

```diff
@@ -95,7 +95,6 @@ public function endTestSuite(TestSuite $suite): void
                 }
 
                 $broker = new Broker($brokerConfig);
-                $broker->createVersionTag();
```

Comment:
Removed because:
1. Tag will be created by `$broker->publish();` line below if provided.
2. If the the version of consumer didn't exist in Pact Broker before, this line will cause this error:
```
Could not create tag. Version [VERSION_HERE] of [CONSUMER_NAME_HERE] does not exist.
```